generic_config_updater/gu_common.py

```diff
@@ -162,6 +166,32 @@ def validate_field_operation(self, old_config, target_config):
                 if any(op['op'] == operation and field == op['path'] for op in patch):
                     raise IllegalPatchOperationError("Given patch operation is invalid. Operation: {} is illegal on field: {}".format(operation, field))
 
+        def _invoke_validating_function(cmd):
+            # cmd is in the format as <package/module name>.<method name>
+            method_name = cmd.split(".")[-1]
+            module_name = ".".join(cmd.split(".")[0:-1])
+            module = importlib.import_module(module_name, package=None)
+            method_to_call = getattr(module, method_name)
+            return method_to_call()
+
+        if os.path.exists(GCU_TABLE_MOD_CONF_FILE):
+            with open(GCU_TABLE_MOD_CONF_FILE, "r") as s:
+                gcu_table_modification_conf = json.load(s)
+        else:
+            raise GenericConfigUpdaterError("GCU table modification validators config file not found") 
+
+        for element in patch:
+            path = element["path"]
+            table = re.search(r'\/([^\/]+)(\/|$)', path).group(1) # This matches the table name in the path, eg. PFC_WD without the forward slashes
```

Comment:
>group [](http://example.com/codeflow?start=57&length=5)

Better be cautious about mismatch. Check matching result before getting group item.

Reply:
We should not be able to reach this part of the code if there is a mismatch.

When creating the jsonpatch before this part of the code is reached, the path must be a string that matches the defined regex. Else, an error will be thrown. 
https://github.com/sonic-net/sonic-utilities/blob/master/config/main.py#L1391

I did add extra code to check for mismatch again if that would be more complete.